src/django/api/facilities_download_view_set.py

```diff
@@ -84,3 +111,27 @@ def __can_user_download_over_limit(
             return False
 
         return True
+
+    @staticmethod
+    def reset_facility_download_limits_next_month(
+        facility_download_limit,
+    ):
+        facility_download_limit.last_download_time = timezone.now()
+        facility_download_limit.download_count = 0
+        facility_download_limit.save()
+
+    @staticmethod
+    def is_same_month_as_last_download(last_download_time) -> bool:
+        if not last_download_time:
+            return False
+
+        current_month = timezone.now().month
+        last_download_month = last_download_time.month
+
+        return current_month == last_download_month
+    @staticmethod
+    def update_facility_download_limit(facility_download_limit):
```

Comment:
We should probably prefix this method with `__` to indicate that it's a private method.